roboto/bot.py

```diff
@@ -1518,6 +1519,25 @@ async def get_my_commands(self) -> List[BotCommand]:
             await make_request(self.session, HTTPMethod.POST, '/getMyCommands'),
         )
 
+    async def delete_message(
+        self, chat_id: Union[ChatID, str], message_id: MessageID
```

Comment:
Please leave a trailing comma (we should eventually review everything to make sure we are sticking to that convention, `black` seems a little too flexible on this).

```suggestion
        self, chat_id: Union[ChatID, str], message_id: MessageID,
```